src/ui/src/js/controllers/command_publishing_blocklist.js

```diff
@@ -0,0 +1,109 @@
+import template from '../../templates/add_command_publishing_blocklist.html';
+
+CommandPublishingBlocklistController.$inject = [
+  '$scope',
+  '$rootScope',
+  'DTOptionsBuilder',
+  'CommandPublishingBlocklistService',
+  '$uibModal',
+];
+
+/**
+ * systemIndexController - Controller for the system index page.
+ * @param  {Object} $scope         Angular's $scope object.
+ * @param  {Object} $rootScope     Angular's $rootScope object.
+ * @param  {Object} DTOptionsBuilder
+ * @param  {Object} CommandPublishingBlocklistService command publishing blocklist service object
+ * @param  {$scope} $uibModal     Angular UI's $uibModal object.
+ */
+export default function CommandPublishingBlocklistController(
+    $scope,
+    $rootScope,
+    DTOptionsBuilder,
+    CommandPublishingBlocklistService,
+    $uibModal,
+) {
+  $scope.data = [];
+
+  $scope.dtOptions = DTOptionsBuilder.newOptions()
+      .withOption('autoWidth', false)
+      .withOption(
+          'pageLength',
+          10,
+      )
+      .withOption('order', [
+        [0, 'asc'],
+        [1, 'asc'],
+        [2, 'asc'],
+        [3, 'asc'],
+      ])
+      .withLightColumnFilter({
+        0: {
+          html: 'input',
+          type: 'text',
+          attr: {class: 'form-inline form-control', title: 'Namespace Filter', style: 'height: 1%'},
+        },
+        1: {
+          html: 'input',
+          type: 'text',
+          attr: {class: 'form-inline form-control', title: 'System Filter', style: 'height: 1%'},
+        },
+        2: {
+          html: 'input',
+          type: 'text',
+          attr: {class: 'form-inline form-control', title: 'Commands Filter', style: 'height: 1%'},
+        },
+        3: {
+          html: 'input',
+          type: 'text',
+          attr: {class: 'form-inline form-control', title: 'Status Filter', style: 'height: 1%'},
+        },
+      })
+      .withBootstrap();
+
+  $scope.deleteBlocklistEntry = function(blockedCommandId, index) {
+    CommandPublishingBlocklistService.
+        deleteCommandPublishingBlocklist(blockedCommandId).then(
```

Comment:
There's currently no feedback when this delete fails.  There should be one of those red error banners at the top of the page similar to what happens on the garden admin page when the save fails.